src/processors/knex-processor.ts

```diff
@@ -73,7 +70,7 @@ export default class KnexProcessor<
       .select(...attributes, "id")
       .modify(queryBuilder => this.optionsBuilder(queryBuilder, op));
 
-    return this.convertToResources(type, records);
+    return records;
```

Comment:
Convert to resources is now a concern of the base class operation processor and is applied to all operations via `execute`.